net/vallumd/files/vallumd.conf

```diff
@@ -0,0 +1,5 @@
+config vallumd mqtt
+	option host 127.0.0.1
+	option port 1883
+	list topics blacklist4
+	list topics blacklist6
```

Comment:
I do not run test this yet, but it will be convenient if it's a default off config